generator/app/template/package.json

```diff
@@ -3,25 +3,31 @@
   "version": "0.0.1",
   "description": "app1 microservice",
   "scripts": {
-    "start": "node ./dist/app.js",
-    "test": "mocha ./dist/test",
+    "build": "babel src -d dist --source-maps",
```

Comment:
Agree. Unfortunately, however, the documentation is completely out of date anyway. I'll make a separate ticket to update it.